pkg/hs/urbit-king/lib/Urbit/Vere/Eyre.hs

```diff
@@ -270,6 +271,10 @@ startServ multi who isFake conf plan = do
       fil = pierPath <> "/.http.ports"
 
   logDebug $ displayShow ("EYRE", "All Servers Started.", srvId, por, fil)
+  for secPor $ \p ->
+    stderr ("http: secure web interface live on https://localhost:" <> tshow p)
```

Comment:
You probably don't want to mention "localhost" in these. Vere doesn't and most of the time, a user won't be accessing it via 127.0.0.1. In the case of the secure web interface, it's wrong since the SSL certificate is extremely unlikely to match "localhost".

Reply:
I don't love these messages, but it looks to me like they match #3092

Reply:
Ah, it looks like I was looking at an out-dated master locally. Up to you ~pilfer if you want to make this change.

Reply:
Joe's right that I just copied verbatim the behavior of C here (except for adding the word "secure" in the https case), including the fact that the text "localhost" is hardcoded. I agree that it's a bit weird.

I'm happy to do whatever you guys want here; just let me know what it is.

Reply:
If it is, somehow, easy to know what IP/domain we're available on, do put that in there.

The reason I put "localhost" in #3092 is because I just wanted to fix the complaints from https://github.com/urbit/urbit/issues/1332#issuecomment-513029707, and figuring those things out was much more involved than I cared for. If you're not running locally, you're likely to understand the implications of "localhost" here.